addons/website_sale_delivery_mondialrelay/controllers/controllers.py

```diff
@@ -73,6 +73,10 @@ def _update_website_sale_delivery_return(self, order, **post):
 
         return res
 
+    def _get_carriers_express_checkout(self, order):
+        res = super()._get_carriers_express_checkout(order)
+        return {carrier: price for carrier, price in res.items() if not carrier.is_mondialrelay}
+
```

Comment:
As Mondial relay doesn't have a `rate_shipment` method, this override is not needed because `rate['success']` will never exist :)